src/ParallelAlgorithms/Events/ObserveFields.hpp

```diff
@@ -124,8 +125,37 @@ class ObserveFields<VolumeDim, ObservationValueTag, tmpl::list<Tensors...>,
         "on a new mesh.";
   };
 
+  /// The floating point type/precision with which to write the data to disk.
+  ///
+  /// Must be specified once for all data or individually for each variable
+  /// being observed.
+  struct FloatingPointTypes {
```

Comment:
Re https://github.com/sxs-collaboration/spectre/pull/3239#discussion_r646040722:

Here's a suggestion to make the user-interface friendlier with minimal changes:

1. Give this tag and the one below a `std::string name()` ("Fields" and "Coordinates") and put them in an option group named "Precision".
2. In `operator<<` for `FloatingPointType` change "Float" -> "Single"
3. Update the input files to:
    ```yaml
    Precision:
      Coordinates: Single
      Fields: [Double]
    ```

This doesn't require renames in the code but hopefully makes the input files easier to understand for users.

Reply:
It's the input file changes that are the most tedious... There are 9 input files currently that need to be updated with different indentation, etc. so basically _any_ rename is the same amount of work...

Reply:
Since copy-pasting the yaml snippet above into 9 input files takes a few minutes at most (10 minutes max including the code changes) and hopefully makes the input files easier to understand for users, my suggestion is to just make this change..

Reply:
Thanks for the suggestion! I can't allocate the time to making this change, I spent a decent amount of time rebasing and so can't spend more time on it.